### PR TITLE
OCPBUGS-28685: Remove openshift-sdn-based Installer CI jobs

### PR DESCRIPTION
**Pull request by sadasu**

### CODE REVIEW

**Comment by sadasu** [comment]
/pj-rehearse pull-ci-openshift-installer-master-e2e-metal-ipi-ovn pull-ci-openshift-installer-master-e2e-metal-ipi-ovn-swapped-hosts pull-ci-openshift-installer-master-e2e-openstack-ovn-parallel

**Comment by sadasu** [comment]
/pj-rehearse pull-ci-openshift-installer-master-e2e-metal-ipi-ovn pull-ci-openshift-installer-master-e2e-metal-ipi-ovn-swapped-hosts pull-ci-openshift-installer-master-e2e-openstack-ovn-parallel

File: ci-operator/config/openshift/installer/openshift-installer-master.yaml [hunk 1]
```diff
@@ -494,16 +494,14 @@ tests:
       TEST_SUITE: experimental/reliability/minimal
     workflow: openshift-e2e-openstack-upi
   timeout: 6h0m0s
-- as: e2e-openstack-sdn-parallel
+- as: e2e-openstack-ovn-parallel
```
Comment (MaysaMacedo) [inline ci-operator/config/openshift/installer/openshift-installer-master.yaml:497]: ditto

**Comment by sadasu** [comment]
@honza , @dtantsur are you OK with the changes for the e2e metal jobs?

**Comment by sadasu** [comment]
/pj-rehearse pull-ci-openshift-installer-master-e2e-metal-ipi-ovn pull-ci-openshift-installer-master-e2e-metal-ipi-ovn-swapped-hosts

**Comment by MaysaMacedo** [comment]
LGTM for the OpenStack jobs

**Review by r4f4 — COMMENTED** [review]
/lgtm

**Comment by sadasu** [comment]
/pj-rehearse pull-ci-openshift-installer-master-e2e-metal-ipi-ovn pull-ci-openshift-installer-master-e2e-metal-ipi-ovn-swapped-hosts

**Comment by sadasu** [comment]
/pj-rehearse pull-ci-openshift-installer-master-e2e-metal-ipi-ovn

**Comment by sadasu** [comment]
/pj-rehearse pull-ci-openshift-installer-master-e2e-metal-ipi-ovn

**Pull request merged**